osf/utils/datetime_aware_jsonfield.py

```diff
@@ -12,13 +12,23 @@
 from psycopg2.extras import Json
 
 
+class NaiveDatetimeException(BaseException):
```

Comment:
I know you've told me that `BaseException` thing before. I always forget and then assume that it would be the one you'd think it would be... Fixed both of those, merging.